JenkinsConsoleUtility/jcuSrc/JUnitXml.cs

```diff
@@ -82,10 +82,14 @@ private static void ParseElementStart(XmlReader reader, bool isEmptyElement)
                     {
                         name = reader.GetAttribute("name")
                     };
-                    int.TryParse(reader.GetAttribute("errors"), out _curSuiteReport.errors);
-                    int.TryParse(reader.GetAttribute("tests"), out _curSuiteReport.tests);
-                    int.TryParse(reader.GetAttribute("failures"), out _curSuiteReport.failures);
-                    int.TryParse(reader.GetAttribute("skipped"), out _curSuiteReport.skipped);
+
+                    int parsedTests = 0;
```

Comment:
Can you explain the change to the TryParse calls here? Why the change to local variable and why get rid of the call for "errors"?

Reply:
errors no longer exists in the public PlayFabAllSDK UUtil class (unit testing class, so only tests are affected, no end product)